src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -796,6 +806,32 @@ def process_string_input(part_name, parsed_question, data_dict):
 
     return replace_tags(html)
 
+
+def process_workspace(part_name, parsed_question, data_dict):
+    """Processes markdown format of workspace questions and returns PL HTML
+    Args:
+        part_name (string): Name of the question part being processed (e.g., part1, part2, etc...)
+        parsed_question (dict): Dictionary of the MD-parsed question (output of `read_md_problem`)
+        data_dict (dict): Dictionary of the `data` dict created after running server.py using `exec()`
+
+    Returns:
+        html: A string of HTML that is part of the final PL question.html file.
+    """
+    if "pl-customizations" in parsed_question["header"][part_name]:
+        if len(parsed_question["header"][part_name]["pl-customizations"]) > 0:
+            raise ValueError("pl-customizations are not supported for workspace questions")
```

Comment:
Same as the other validation for workspaces - its technically not needed, so I'm more than happy to remove it, but it seemed like a good idea to make sure people don't try to do things here, and get confused when they don't work

Reply:
I think this is a good idea, more so to keep the md files readable rather than help developers with debugging. Perhaps this should be part of a pre-commit hook, but that might be a little much.